UnitTests/View/Layout/AbsoluteLayoutTests.cs

```diff
@@ -321,4 +321,69 @@ public void AbsoluteLayout_LayoutSubviews ()
 		Assert.Equal (new Rect (10, 10, 10, 10), v2.Frame);
 		super.Dispose ();
 	}
+
+	[Fact]
+	public void AbsoluteLayout_Setting_Bounds_Location_NotEmpty ()
+	{
+		// TODO: Should we enforce Bounds.X/Y == 0? The code currently ignores value.X/Y which is
+		// TODO: correct behavior, but is silent. Perhaps an exception?
```

Comment:
I think Bounds is very tied to the Margin, Border and Padding thickness. So, the calculation from Frame to Bounds or Bounds to Frame will always return the same values for each situation. I know you have on minds to use negative Bounds location for scrolling purposes by using the Padding to accommodate the scroll bars. But I think that will never work like that without using a container with different location and size, inside the Bounds. But as you always think far ahead, you may already have some idea about this subject.